opinionstage-polls.php

```diff
@@ -1,38 +0,0 @@
-<?php
```

Comment:
suggest we remove this commit, it's dangerous (let's approach this problem in separation from release)